wormhole-connect/src/views/Bridge/Preview.tsx

```diff
@@ -182,8 +81,34 @@ function Preview(props: { collapsed: boolean }) {
     receiveNativeAmt,
     gasEst,
     dispatch,
+    relayerFee,
   ]);
 
+  useEffect(() => {
+    const computeRelayerFee = async () => {
```

Comment:
Wondering if this should be in the main bridge component...

Reply:
I thought so as well, now that it is decoupled from the row building